config/deploy.rb

```diff
@@ -25,7 +25,7 @@
 
 # Default value for linked_dirs is []
 # set :linked_dirs, %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
-set :linked_dirs, %w(log log/wfs run config/environments config/certs jar tmp)
+set :linked_dirs, %w(log run config/environments config/certs jar tmp)
```

Comment:
I was thinking this was a way to ensure the log/wfs  directory existed at deploy time??